docs/marketplace/add_to_marketplace/contributing_overview.md

```diff
@@ -445,6 +445,39 @@ Here are listed some useful advice to strengthen your items:
 - **Test**: each service must have well-tested code;
 - **Logs**: each service should display the logs, to inform users about the actions they are currently performing and if any errors have been found during their execution.
 
+## Tips & Tricks
+
+ - **Where to host the zip with the code**: If you code your template on github you can access the zip file using the url https://github.com/&lt;username&gt;/&lt;project-name&gt;/archive/&lt;branch&gt;.tar.gz
+ - **.mia-platform folder**: This folder is ignored withn creating the repository, use it to store the images and the marketplace item configuration
```

Comment:
Actually the directory is not ignored but used to provide specific files that are used to overwrite those in the main directory (e.g. the README). Then it won't be commited to the final repository, if this is what you meant I'd try and rephrase like


```suggestion
 - **.mia-platform folder**: This folder is used to overwrite files in the main directory and won't be commited in the created repository; you can use it to provide a different README file to developers or to store the images and the marketplace item configuration)
```